pkg/operation/botanist/monitoring.go

```diff
@@ -103,12 +106,49 @@ func (b *Botanist) DeploySeedMonitoring(ctx context.Context) error {
 	// Need stable order before passing the dashboards to Grafana config to avoid unnecessary changes
 	kutil.ByName().Sort(existingConfigMaps)
 
+	// Apply controlplane user exposed pods
+	cpObservedPods := []ObservedPods{
+		{PodPrefix: "kube-apiserver", UserExposed: true},
+		{PodPrefix: "kube-controller-manager", UserExposed: true},
+		{PodPrefix: "kube-scheduler", UserExposed: true},
+		{PodPrefix: "cluster-autoscaler", UserExposed: true},
+	}
+
+	if b.Shoot.WantsVerticalPodAutoscaler {
+		vpaObservedPods := []ObservedPods{
+			{PodPrefix: "vpa-admission-controller", UserExposed: true},
+			{PodPrefix: "vpa-recommender", UserExposed: true},
+			{PodPrefix: "vpa-updater", UserExposed: true},
+		}
+
+		cpObservedPods = append(cpObservedPods, vpaObservedPods...)
+	}
+
+	usersObservedPods = append(usersObservedPods, cpObservedPods...)
+	operatorsObservedPods = append(operatorsObservedPods, cpObservedPods...)
+
 	// Read extension monitoring configurations
 	for _, cm := range existingConfigMaps.Items {
 		alertingRules.WriteString(fmt.Sprintln(cm.Data[v1beta1constants.PrometheusConfigMapAlertingRules]))
 		scrapeConfigs.WriteString(fmt.Sprintln(cm.Data[v1beta1constants.PrometheusConfigMapScrapeConfig]))
 		operatorsDashboards.WriteString(fmt.Sprintln(cm.Data[v1beta1constants.GrafanaConfigMapOperatorDashboard]))
 		usersDashboards.WriteString(fmt.Sprintln(cm.Data[v1beta1constants.GrafanaConfigMapUserDashboard]))
+
+		components := cm.Data["observedComponents"]
```

Comment:
is this new contract/format documented in `./docs/extensions/logging-and-monitoring.md`? Does it make sense to update the docs with the changes from this PR?